pages/studentPortal/web/React/tailwindcss-colourstyling.tsx

```diff
@@ -0,0 +1,70 @@
+import { useRouter } from "next/router";
+import React from "react";
+import LessonComponent, {
+  LessonComponentData,
+} from "../../../../components/coding/studentPortal/LessonComponent";
+import { Button } from "../../../../components/ui/Button";
+
+const TailwindColourStyling = ({ lessonComponents }) => {
+  const router = useRouter();
+
+  const handleContinue = () => {
+    router.push("/studentPortal/web/React/tailwindcss-gridflex");
+  };
+  return (
+    <>
+      <div className="grid grid-cols-1 gap-8 px-4 pt-4 m-8 sm:px-12">
+        {lessonComponents.map((it, index) => (
+          <LessonComponent data={it} key={index} />
+        ))}
+      </div>
+      <div className="flex my-8 mr-8 sm:justify-end">
+        <Button onClick={handleContinue} label="Continue" />
+      </div>
+    </>
+  );
+};
+
+export async function getServerSideProps({ params }) {
+  const lessonComponents: LessonComponentData[] = [
```

Comment:
can we add a resource-list here with links to the specific sections (bg-color and font)